From e69a2ca59d32af24beef8f1d26cecd81f1db8560 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Tue, 22 May 2018 10:54:05 +0800
Subject: [PATCH 3859/5242] MLK-18365: dts: change hdmi dig_pll clock rate to
 675MHz

commit  cf503f60a7d71212098bdb2f04f08ae321c8c27e from
https://source.codeaurora.org/external/imx/linux-imx.git

Change the hdmi dig_pll clock rate to 675MHz,
hdmi core clock is source from dig_pll.
And HDMI CEC required core clock should integer MHz(675/5=135MHz).

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 99709d8..671080d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -2681,7 +2681,7 @@
 		clock-names = "ipg";
 		assigned-clocks = <&clk IMX8QM_HDMI_DIG_PLL_CLK>,
 							<&clk IMX8QM_HDMI_LIS_IPG_CLK>;
-		assigned-clock-rates = <1188000000>, <85000000>;
+		assigned-clock-rates = <675000000>, <85000000>;
 		power-domains = <&pd_hdmi>;
 	};
 
-- 
1.7.9.5

